examples/user_guide/Interactive.ipynb

```diff
@@ -92,6 +123,68 @@
     "ds.air.interactive(width=800).sel(time=pnw.DiscreteSlider)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Functions as inputs\n",
+    "\n",
+    "In some cases your starting point for your interactive pipeline may not simply be a DataFrame or xarray Dataset but a function that fetches some data or applies some initial processing on your data. In such a case you can use the `hvplot.bind` function to bind static AND dynamic arguments to your function. Binding dynamic arguments such as a widget or parameter means that whenever the widget/parameter value changes the output of the function will change as well. This makes it possible to construct functions as the input to your interactive pipeline that themselves represent some data pipeline.\n",
```

Comment:
What return types are supported? Dask and Pandas Dataframes and Xarray Datasets?

Reply:
gpd.DataFrame, cudf.DataFrame, dask.dataframe.DataFrame, pandas.DataFrame.